packages/expo-image-picker/ios/EXImagePicker/EXImagePicker.m

```diff
@@ -40,8 +40,9 @@ - (instancetype)init
 {
   if (self = [super init]) {
     self.defaultOptions = @{
-                            @"allowsEditing" : @NO,
+                            @"allowsEditing": @NO,
                             @"base64": @NO,
+                            @"exportPreset": @"Passthrough",
```

Comment:
There are two places now where we define _default_ value. Here and in `mapPreset:` function. Maybe we could drop one of the places? (I would say `mapPreset:` `if` guards for more invalid situations.)